diff --git a/physical/etcd/etcd3.go b/physical/etcd/etcd3.go
index 94e617997..c03698582 100644
--- a/physical/etcd/etcd3.go
+++ b/physical/etcd/etcd3.go
@@ -84,13 +84,22 @@ func newEtcd3Backend(conf map[string]string, logger log.Logger) (physical.Backen
 	cert, hasCert := conf["tls_cert_file"]
 	key, hasKey := conf["tls_key_file"]
 	ca, hasCa := conf["tls_ca_file"]
+	sskip, hasSkip := conf["tls_insecure_skip_verify"]
 	if (hasCert && hasKey) || hasCa {
 		tls := transport.TLSInfo{
 			TrustedCAFile: ca,
 			CertFile:      cert,
 			KeyFile:       key,
 		}
-
+		if hasSkip {
+			skip, err := strconv.ParseBool(sskip)
+			if err != nil {
+				return nil, fmt.Errorf("value of 'tls_insecure_skip_verify' (%v) could not be understood", sskip)
+			}
+			tls.InsecureSkipVerify = skip
+		} else {
+			tls.InsecureSkipVerify = false
+		}
 		tlscfg, err := tls.ClientConfig()
 		if err != nil {
 			return nil, err
